milabench/utils.py

```diff
@@ -102,3 +108,92 @@ def make_constraints_file(pth, constraints):
         return (pth,)
     else:
         return ()
+
+
+def discover_validation_layers(module):
+    """Discover validation layer inside the milabench.validation module"""
+    path = module.__path__
+    name = module.__name__
+
+    layers = {}
+
+    for _, layerpath, _ in pkgutil.iter_modules(path, name + "."):
+        layername = layerpath.split(".")[-1]
+        layermodule = importlib.import_module(layerpath)
+
+        if hasattr(layermodule, "_Layer"):
```

Comment:
A single leading underscore typically denotes a private variable. I would make it `Layer` or `__layer__` or `__layers__ = {"layer_name": LayerClass}` if we want to allow a module to export multiple layers.

Reply:
done, renamed to Layer and added the possibility to specify __layers__ as well